docs/python/how-to-guides/use-filters.md

```diff
@@ -193,6 +193,22 @@ iris_slice_pct = iris.slice_pct(0.45, 0.55)
 iris_tail_pct = iris.tail_pct(0.1)
 ```
 
+### Incremental Release Filter
+
+The incremental release filter, converts a static or add-only table into a ticking table that parcels out rows over time. This can be useful to simulate ticking data for development or to limit the number of rows that a complex query processes at one time. The incremental release filter takes two parameters:
```

Comment:
```suggestion
The incremental release filter converts a static or add-only table into a ticking table that parcels out rows over time. This can be useful to simulate ticking data for development or to limit the number of rows that a complex query processes at one time. The incremental release filter takes two parameters:
```